src/shared/components/form/formSelect/formSelect.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class FormSelect extends Component {
+  buildOptions = () => {
+    const { prompt, options } = this.props;
+    const opts = [];
+
+    if (prompt) {
+      opts.push(<option key="prompt" value="">{ prompt }</option>);
+    }
+    options.forEach(option => opts.push(<option key={option.value} value={option.value}>{option.label}</option>));
+    return opts;
+  }
+
+  render() {
+    return (
+      <select id={this.props.id} onChange={this.props.onChange}>
+        {this.buildOptions()}
+      </select>
+    );
+  }
```

Comment:
^ Actually, it appears as if the JS is working correctly as is.  The reason that the selected option is not being displayed (I'm using Chrome on Windows) appears to be from the CSS padding styling being applied to the select elements. See comments on the CSS styling for the selects in mentorRequest.css and signup.css.